docker-test.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -vex
+
+repo_path=$GOPATH/src/github.com/taskcluster
+
+cd $repo_path
+go get ./...
```

Comment:
why `go get everything under taskcluster`?

Reply:
It is a bug, thanks for pointing this out